src/components/Navbar/Navbar.js

```diff
@@ -1,35 +1,50 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { Icon, Button } from 'antd';
+import Menu from 'antd/lib/menu';
 
-const NavBar = ({ currentUser: { isAuthenticated, user }, logoutUser }) => (
-  <ul>
-    {
-      isAuthenticated === false &&
-        <li>
-          <NavLink to="/">
-            Home
+class NavBar extends React.Component {
+  render() {
+    const { currentUser: { isAuthenticated, user }, logoutUser } = this.props;
+    return (
+      <Menu
+        mode="horizontal"
+      >
+        {
+          isAuthenticated === false &&
+          <Menu.Item key="home">
+            <Icon type="home" />
+            <NavLink to="/">
+              Home
+            </NavLink>
+          </Menu.Item>
+        }
+        {
+          isAuthenticated ?
+            <Menu.Item key="logout">
```

Comment:
Declare these at their own stateless functional components. can be in the same file if only used here. Will make your code much easier to read. something like:
`
isAuthenticated ? <LogoutMenuItem /> : <SignUpOrLoginMenuItem />
`